google-cloud-logging/lib/google/cloud/logging/project.rb

```diff
@@ -347,22 +376,17 @@ def async_writer max_queue_size: AsyncWriter::DEFAULT_MAX_QUEUE_SIZE
         #   gcloud = Google::Cloud.new
         #   logging = gcloud.logging
         #
-        #   resource = logging.resource "gae_app", labels: {
-        #                                 "module_id" => "1",
-        #                                 "version_id" => "20150925t173233" }
-        #                               }
-        #
-        #   logger = logging.logger "my_app_log", resource,
-        #                           labels: {env: :production}
-        #   logger.info "Job started."
-        #
-        def logger log_name, resource, labels: {}, async_writer: true
-          async_writer = case async_writer
-                         when true then self.async_writer
-                         when false then nil
-                         else async_writer
-                         end
-          Logger.new self, log_name, resource, labels, async_writer
+        #   resource = logging.resource "gae_app",
+        #                               module_id: "1",
+        #                               version_id: "20150925t173233"
+        #
+        #   logger = logging.async_logger "my_app_log",
+        #                                 resource,
+        #                                 env: :production
+        #   logger.info "Job started." # asynchronous call
+        #
+        def async_logger log_name, resource, labels = {}
```

Comment:
I can see adding `max_queue_size` to the method arguments since it is specific to the async_writer object that will be created. That argument would not make sense on the default `logger` method, and is part of why I prefer this solution.